content/docs/catalog/rich-team-pages/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Creating rich team pages
+publishedDate: '2023-10-05'
```

Comment:
ooohh no time.. I must try this.

Reply:
Dont tell anyone, but I had been copying the publishedDate from another file and changing the date part.. :/